.devcontainer/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM rocker/r-base
+
+RUN apt-get -qq update \
+  && apt-get install -y --no-install-recommends \
+  git libxml2-dev
+
+COPY DESCRIPTION .
+
+# NB: patrick also pulls in testthat and pkgload
```

Comment:
This comment is no longer relevant, right?

Reply:
right; axed